### PR TITLE
Updated user documentation

### PR DESCRIPTION
**Pull request by infohash**
Addresses #120

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -88,7 +200,7 @@ def logout():
 ```
 
 If the logout view is mounted under a custom endpoint (other than the default, which is 
-[the name of the view function](http://flask.pocoo.org/docs/1.0/api/#flask.Flask.route)), or if using Blueprints, you
+[the name of the view function](https://flask.palletsprojects.com/en/2.0.x/api/#flask.Flask.route)), or if using Blueprints, you
```
Comment (zamzterz) [inline docs/quickstart.md:203]: 💯

**Review by zamzterz — CHANGES_REQUESTED** [review]
Really nice! 👏 

Most comments are minor, but there are some structural questions (on my part) and the mentioning of "keycloak" which should be replaced. 🙂

File: docs/configuration.md [hunk 2]
```diff
@@ -1,33 +1,80 @@
...
+provider_metadata = ProviderMetadata(issuer='https://keycloak:8080/auth/realms/master',
+                                     authorization_endpoint='https://keycloak:8080/auth/realms/master/protocol/openid-connect/auth',
+                                     token_endpoint='https://keycloak:8080/auth/realms/master/protocol/openid-connect/token',
+                                     introspection_endpoint='https://keycloak:8080/auth/realms/master/protocol/openid-connect/token/introspect',
+                                     userinfo_endpoint='https://keycloak:8080/auth/realms/master/protocol/openid-connect/userinfo',
+                                     end_session_endpoint='https://keycloak:8080/auth/realms/master/protocol/openid-connect/logout',
+                                     jwks_uri='https://keycloak:8080/auth/realms/master/protocol/openid-connect/certs',
+                                     registration_endpoint='https://keycloak:8080/auth/realms/master/clients-registrations/openid-connect'
+                                     )
+# As shown earlier, if you are using Static Client Configuration, then specify
```
[the remark this answers is not in the document]
Comment (infohash) [reply docs/configuration.md:69]: They both are arguments of [provider metadata](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata).

**Comment by zamzterz** [comment]
The failing tests were due to the removed `example/__init__.py` file - I've fixed it up.
Thanks 👏

**Review by zamzterz — APPROVED** [review]

**Pull request merged**